pipelines/templates/build-python-template.yaml

```diff
@@ -0,0 +1,91 @@
+# This template builds a python wheel and container and pushes them to Azure Artifcats 
+# and a cotainer registry, using presidio Makefile.
+# It is used in CI to build the analyzer container.
+# parameters:
+#   folder_name: where the python files are located.
+#   package_name: name of python package to build
+#   registry_service_connection: service connection to docker registry.
+#   registry_name: full name of container registry (i.e. presidio.azurecr.io)
+#   presidio_label: image label.
+#   deps_label: label of deps container used when building this container.
+#   artifact_feed: pypi artifact used to push the wheel
+parameters:
+  folder_name: '' 
+  package_name: ''
+  artifact_feed: ''
+  presidio_label: ''
+  deps_label: ''
+  registry_name: ''
+  registry_service_principal: ''
+steps:
+- task: Docker@2
+  displayName: 'Docker login'
+  inputs:
+    containerRegistry: ${{ parameters.registry_service_principal }}
+    command: 'login'
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '$(PYTHON.VERSION)'
+  displayName: 'Use Python $(PYTHON.VERSION)'
+- script: |
+    python -m pip install --upgrade pip
+    python -m pip install pipenv
+    pipenv --python 3
+    pip install twine
+  displayName: 'Install python environment'
+  workingDirectory: ${{ parameters.folder_name }}
+- task: TwineAuthenticate@0
+  inputs:
+    artifactFeeds: ${{ parameters.artifact_feed }}
+  displayName: 'Authenticate RC with Azure Artifacts'
```

Comment:
minor, can we have the display name the first item? easier to read

Reply:
fixed